gaia-header.js

```diff
@@ -278,9 +278,9 @@ module.exports = component.register('gaia-header', {
   setTitleStyle: function(el, style) {
     debug('set title style', style);
     this.observerStop();
-    el.style.MozMarginStart = style.marginStart + 'px';
-    el.style.MozPaddingStart = style.padding.start + 'px';
-    el.style.MozPaddingEnd = style.padding.end + 'px';
+    el.style.marginLeft = style.marginStart + 'px';
+    el.style.paddingLeft = style.padding.start + 'px';
+    el.style.paddingRight = style.padding.end + 'px';
```

Comment:
please add a comment to explain we should use MozPaddingStart and friends once we make gaia-header RTL-compatible. (not sure about MozMarginStart given latest mail from Jim Porter on dev-gaia though :/)